python/bind_operators.h

```diff
@@ -20,6 +20,28 @@ void bind_comparison(pybind11::class_<T, Ignored...> &c) {
       py::call_guard<py::gil_scoped_release>());
 }
```

Comment:
The function above should probably not exist any more, or be replaced by the new one your write below?